src/sentry/integrations/services/integration/impl.py

```diff
@@ -405,8 +411,8 @@ def send_incident_alert_notification(
                 return False
 
             app_platform_event = AppPlatformEvent(
-                resource="metric_alert",
-                action=new_status_str,
+                resource=SentryAppResourceType.METRIC_ALERT,
+                action=SentryAppActionType(new_status_str),
```

Comment:
Does this need to be using a specific type for data? I see that `AppPlatformEvent` takes a generic that's used to type the `data` field